docs/site/Transactions.md

```diff
@@ -23,10 +23,10 @@ is attached to one of the following connectors:
 - [PostgreSQL connector](PostgreSQL-connector.html)
 - [SQL Server connector](SQL-Server-connector.html)
 - [Oracle connector](Oracle-connector.html)
-- [DB2 Connector](DB2-connector.html)
 - [DashDB Connector](DashDB.html)
-- [DB2 iSeries Connector](DB2-iSeries-connector.html)
-- [DB2 for z/OS connector](DB2-for-z-OS.html)
+- [IBM Db2 (for Linux, Unix, Windows) Connector](DB2-connector.html)
+- [IBM Db2 for i connector](DB2-for-i.html)
+- [IBM Db2 for z/OS connector](DB2-for-z-OS.html)
```

Comment:
These links are broken. They should be `(xxxx.md)` instead of `.html`. Sorry that we didn't notice that before.